LOC-25114 - Recommendation list creation documentation (#811)
vtexdocs/help-center-content: docs/es/tutorials/beta/recomendaciones-de-productos-beta/crear-lista-de-recomendaciones.md

2. Haz clic en el botón `+ Lista`.

>    ![criar-lista-de-recomendacoes-gif](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/beta/recomendações-de-produtos-beta/criar-lista-de-recomendacoes_1.gif)

Reviewer: ```suggestion
   ![crear-lista-de-recomendaciones-gif](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/es/tutorials/beta/recomendaciones-de-productos-beta/crear-lista-de-recomendaciones_1.gif)
```

   - **Nombre:** nombre que deseas dar a la lista de recomendaciones. Este será el título de la lista mostrado en la tienda. Recomendamos que utilices el título que tendrá la lista en el storefront (por ejemplo: "Para ti").
   - **Tags (opcional):** etiquetas internas que agrupan las listas de recomendación para facilitar el análisis de métricas. Recomendamos que incluyas la página donde se utilizará la lista como tag (por ejemplo: Home). Escribe la tag deseada y presiona `Enter` o la tecla `,` (coma) para agregarla. Puedes agregar múltiples tags.

>    ![criar-lista-de-recomendacoes-2](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/beta/recomendações-de-produtos-beta/criar-lista-de-recomendacoes_2.png)

Reviewer: ```suggestion
   ![crear-lista-de-recomendaciones-2](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/es/tutorials/beta/recomendaciones-de-productos-beta/crear-lista-de-recomendaciones_2.png)
```

   Si la estrategia es **Colección manual**, completa también:

   - **Productos:** nombre de la [colección creada en el catálogo](https://help.vtex.com/es/docs/tutorials/registrar-colecciones-beta), con el ID de la colección entre paréntesis.
   - **Ordenación:** opción para ordenar los productos recomendados:

     - Más vendidos
     - Precio más alto
     - Precio más bajo
     - Nombre A–Z
     - Nombre Z–A
     - Fecha de lanzamiento (más recientes)
     - Mayor descuento

5. Consulta la vista previa de las recomendaciones para validar el resultado antes de guardar.

   Si la estrategia elegida es **Comprados juntos**, **Productos similares** o **Productos visualmente similares**, debes seleccionar un **producto de referencia** para generar la vista previa.

6. Haz clic en el botón `Crear` para guardar la lista de recomendaciones.

   Se mostrará el mensaje de confirmación _Lista creada con éxito_. A partir de ese momento, la lista estará disponible para implementar en el storefront.

>    ![criar-lista-de-recomendacoes-3](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/beta/recomendações-de-produtos-beta/criar-lista-de-recomendacoes_3.png)

Reviewer: ```suggestion
   ![crear-lista-de-recomendaciones-3](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/es/tutorials/beta/recomendaciones-de-productos-beta/crear-lista-de-recomendaciones_3.png)
```

Utiliza el [ID copiado](#copiar-id-de-la-lista) para agregar la estantería de recomendaciones en las páginas de tu tienda.

- Para Store Framework, consulta la documentación del componente [Recommendation Shelf](https://developers.vtex.com/docs/apps/vtex.recommendation-shelf).

> - ParaFastStore o Headless, consulta la guía [Integrating product recommendations in headless or FastStore web stores](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-headless-or-faststore-web-stores).

Reviewer: ```suggestion
- Para FastStore o Headless, consulta la guía [Integrating product recommendations in headless or FastStore web stores](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-headless-or-faststore-web-stores).
```

> - Para tiendas headless, consulta la guía [Integrating product recommendations in headless stores (Beta)](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-headless-stores).

Reviewer: ```suggestion
- Para Store Framework, consulta la documentación del componente [Recommendation Shelf](https://developers.vtex.com/docs/apps/vtex.recommendation-shelf).
- ParaFastStore o Headless, consulta la guía [Integrating product recommendations in headless or FastStore web stores](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-headless-or-faststore-web-stores).
- Para aplicaciones Mobile, consulta la guía [Integrating product recommendations in mobile stores](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-mobile-stores).
```

- [Recomendaciones de productos (beta)](https://help.vtex.com/es/docs/tutorials/recomendaciones-de-productos-beta)
- [Dashboard de recomendaciones (beta)](https://help.vtex.com/es/docs/tutorials/dashboard-de-recomendaciones)
- [Recommendation Shelf (Store Framework)](https://developers.vtex.com/docs/apps/vtex.recommendation-shelf)
- [Implementing product recommendations in FastStore](https://developers.vtex.com/docs/guides/implementing-product-recommendations-in-faststore)

> - [Integrating product recommendations in headless stores (Beta)](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-headless-stores)

Reviewer: ```suggestion
- [Integrating product recommendations in headless or FastStore web stores](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-headless-or-faststore-web-stores)
- [Integrating product recommendations in mobile stores](https://developers.vtex.com/docs/guides/integrating-product-recommendations-in-mobile-stores)
```
